src/View/mainpage/DicomView.py

```diff
@@ -70,21 +70,24 @@ def init_view(self):
         self.view.setBackgroundBrush(background_brush)
     
     def wheelEvent(self, event: QtGui.QWheelEvent):
+        degree = 8
+        step = 15
+        #https://doc.qt.io/qt-6/qwheelevent.html#angleDelta
         modifiers = event.modifiers()
         if modifiers == QtCore.Qt.ShiftModifier:
             super().wheelEvent(event)
         elif modifiers == QtCore.Qt.AltModifier:
             super().wheelEvent(event)
         elif modifiers == QtCore.Qt.ControlModifier:
-            angle = event.angleDelta() / 8
-            delta = angle.y() / 15
+            angle = event.angleDelta() / degree
+            delta = angle.y() / step
             if delta > 0:
                 self.zoom_in()
             else:
                 self.zoom_out()
         else:
-            num_degrees = event.angleDelta() / 8
-            num_steps = num_degrees / 15
+            num_degrees = event.angleDelta() / degree
+            num_steps = num_degrees / step
```

Comment:
and this suggests that the variable has units of degrees per step (or alternatively, that it's inverse has units of steps per degree).
units are sometimes useful in the names of variables to help clarify their meaning.